dinghy-lib/src/apple/device.rs

```diff
@@ -368,11 +375,65 @@ fn make_ios_app(
         .split(" ")
         .last()
         .ok_or_else(|| anyhow!("empty magic"))?;
-    xcode::add_plist_to_app(&build_bundle, target, app_id)?;
+    xcode::add_plist_to_ios_app(&build_bundle, target, app_id)?;
+    Ok(build_bundle)
+}
+
+fn make_watchos_app(
```

Comment:
It seems like these two functions really duplicate a lot of make_ios_app. I think it would be easier to maintain if most of the commonality were factored out or if make_ios_app were renamed and parameterized over `AppleSimulatorType` so that it could handle all three cases.

Reply:
I kinda agree here, unless there is something I can't see.

Reply:
Yeah, I originally thought I might have to do more changes per simulator target so I just duplicated the functions. The bulk of the difference is now in the plist addition.